src/routes/v2/channels.js

```diff
@@ -126,4 +127,41 @@ module.exports = [
             return apps
         },
     },
+    {
+        method: 'DELETE',
+        path: '/v2/channels/{uuid}',
+        config: {
+            auth: 'token',
+            tags: ['api', 'v2'],
+        },
+        handler: async (request, h) => {
+            request.logger.info('In handler %s', request.path)
+
+            if (!currentUserIsManager(request)) {
+                throw Boom.unauthorized()
+            }
+
+            const { uuid } = request.params
+            const knex = h.context.db
+
+            try {
```

Comment:
There's an potential error here if some error is thrown in the try-block, as the transaction is not rolled back.